### PR TITLE
nordnm: add page

### PR DESCRIPTION
**Pull request by PhilippThaler**
https://github.com/Chadsr/NordVPN-NetworkManager

I don't know if many people are using this but I think it's a great tool to set up your NordVPN connections.

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/nordnm.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+> This tool automates the importing and secures the usage of NordVPN OpenVPN configurations through Network Manager.
+
+- View available categories and countries:
+
+`sudo nordnm list --categories --countries`
+
+- Synchronise, update configuration files, activate the kill-switch and auto-connect to a "normal" UDP server in the US:
+
+`sudo nordnm sync -uka us normal udp`
```
[the remark this answers is not in the document]
Comment (PhilippThaler) [reply pages/linux/nordnm.md]: Thanks for helping, I'm pretty new to how this all works :) ^^

File: pages/linux/nordnm.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# nordnm
+
+> This tool automates the importing and secures the usage of NordVPN OpenVPN configurations through Network Manager.
```
Comment (agnivade) [inline pages/linux/nordnm.md]: This is lifted verbatim from the site. We always try to explain it in simpler terms. Otherwise there would be no difference from the original page, which the users can always look up.

How about - "A tool for managing NordVPN OpenVPN configurations" ?

Note that a tldr page is not meant to be exhaustive. Just enough information in a lucid manner to get a beginner started. For complete information, the man page is always there.

File: pages/linux/nordnm.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+- Synchronise, update configuration files, activate the kill-switch and auto-connect to a "normal" UDP server in the US:
+
+`sudo nordnm sync -uka us normal udp`
+
+- View metrics of the synchronised servers:
+
+`sudo nordnm list --active-servers`
+
+- Set your MAC address to be randomised each time you connect to a network:
```
Comment (agnivade) [inline pages/linux/nordnm.md]: Set _the_ MAC address

File: pages/linux/nordnm.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# nordnm
+
+> This tool automates the importing and secures the usage of NordVPN OpenVPN configurations through Network Manager.
+
+- View available categories and countries:
+
+`sudo nordnm list --categories --countries`
+
+- Synchronise, update configuration files, activate the kill-switch and auto-connect to a "normal" UDP server in the US:
```
Comment (agnivade) [inline pages/linux/nordnm.md]: What is the "kill-switch" ? This description is a bit too long. Any chance we can reduce the no. of options ?
